README.md

```diff
@@ -1,5 +1,10 @@
-# opencollective-api
-OpenCollective's API
+# OpenCollective API
+
+[![Circle CI](https://circleci.com/gh/OpenCollective/opencollective-api/tree/master.svg?style=shield)](https://circleci.com/gh/OpenCollective/opencollective-api/tree/master)
+[![Slack Status](https://slack.opencollective.com/badge.svg)](https://slack.opencollective.com)
+[![Gitter chat](https://badges.gitter.im/OpenCollective/OpenCollective.svg)](https://gitter.im/OpenCollective/OpenCollective)
+[![Dependency Status](https://david-dm.org/opencollective/opencollective-api.svg)](https://david-dm.org/opencollective/opencollective-api)
+[![Known Vulnerabilities](https://snyk.io/test/github/opencollective/opencollective-api/badge.svg)](https://snyk.io/test/github/opencollective/opencollective-api)
```

Comment:
I can remove if you want, but that being said, anyone knowledgeable can access that same link anyway. This is more a reminder of what we must now fix.